test/test.js

```
@@ -2,9 +2,11 @@ const clabe = require('../lib/index.js');
 const {
   BANKS,
   BANK_NAMES,
+  CARD_BINS,
```

ESLint

[error] 'CARD_BINS' is assigned a value but never used.

(no-unused-vars)